numba/cuda/decorators.py

```diff
@@ -20,16 +20,16 @@ def jit(func_or_sig=None, argtypes=None, device=False, inline=False, bind=True,
     """
     JIT compile a python function conforming to the CUDA Python specification.
     If a signature is supplied, then a function is returned that takes a
-    function to compile. If
+    function to compile.
 
     :param func_or_sig: A function to JIT compile, or a signature of a function
-       to compile. If a function is supplied, then an :class:`AutoJitCUDAKernel`
-       is returned. If a signature is supplied, then a function which takes a
-       function to compile and returns an :class:`AutoJitCUDAKernel` is
+       to compile. If a function is supplied, then an
+       :class:`numba.cuda.compiler.AutoJitCUDAKernel` is returned. If a
+       signature is supplied, then a function which takes a function to compile
+       and returns an :class:`numba.cuda.compiler.AutoJitCUDAKernel` is
        returned.
```

Comment:
I'm not sure this makes sense?!

Reply:
Indeed, it is gibberish. Will fix.

Reply:
I've cleared this up a bit now - hopefully it is clearer.